### PR TITLE
Remove federation worker count from site settings

### PR DESCRIPTION
**Pull request by cetra3**
Sister PR to:

https://github.com/LemmyNet/lemmy/pull/3158

### CODE REVIEW

**Review by SleeplessOne1917 — APPROVED** [review]
LGTM. I'll merge this after the backend PR is merged.

**Comment by dessalines** [comment]
I got this one merged in now, `0.18.1-rc.1`

**Pull request closed without merging**
